padacioso/opm.py

```
@@ -46,27 +48,26 @@ def __repr__(self):
-        self.padacioso_config = config
-        self.bus = bus
+    def __init__(self, bus: Optional[Union[MessageBusClient, FakeBus]] = None,
+                 config: Optional[Dict] = None):
+        super().__init__(config=config or {}, bus=bus)
 
         core_config = Configuration()
         self.lang = standardize_lang_tag(core_config.get("lang", "en-US"))
         langs = core_config.get('secondary_langs') or []
         if self.lang not in langs:
             langs.append(self.lang)
         langs = [standardize_lang_tag(l) for l in langs]
```

Ruff

Ambiguous variable name: `l`

(E741)